components/kyma-environment-broker/internal/provisioner/client.go

```diff
@@ -25,16 +30,34 @@ type Client interface {
 }
 
 type client struct {
-	graphQLClient *graphql.Client
+	graphQLClient *gcli.Client
 	queryProvider queryProvider
 	graphqlizer   Graphqlizer
+
+	queryDumping bool
 }
 
-func NewProvisionerClient(endpoint string, queryLogging bool) Client {
+func NewProvisionerClient(endpoint string, queryDumping bool) Client {
+	httpClient := &http.Client{
+		Transport: &http.Transport{
```

Comment:
What about the timeout? Right now you have a zero values and
>  Zero means no timeout.

so use `http.DefaultTransport.(*http.Transport).Clone()` and then change the values that you want to
+ add timeout to client itself


------
**EDIT:**

I just discover that we have a pkg for that 
```
httputil.NewClient(30, cfg.Director.SkipCertVerification)
```

but this package we also have a bug about default timeouts in transport layer. Please fix it as described above :)

Reply:
I've changed httputil